docs/sources/panels-visualizations/visualizations/histogram/index.md

```diff
@@ -138,6 +158,10 @@ Transparency of the gradient is calculated based on the values on the Y-axis. Th
 
 Gradient color is generated based on the hue of the line color.
 
+#### Scheme
```

Comment:
I know you were just following the pattern here, but let's fix it while you're working on this. Can you please make these **Gradient mode** options a bullet list like what I've suggested for the **Stacking** options.